Chapter5/stats_handler.erl

```diff
@@ -0,0 +1,13 @@
+-module(stats_handler).
```

Comment:
How do I add `stats_handler` to event manager? Who is responsible for init `Db` state of `stats_handler`?